kani-compiler/src/codegen_cprover_gotoc/codegen/place.rs

```diff
@@ -334,21 +340,15 @@ impl<'tcx> GotocCtx<'tcx> {
     ///
     /// Since the goto representation for both is the same, we use the expected type to decide
     /// what to return.
-    fn codegen_simd_field(
-        &mut self,
-        parent_expr: Expr,
-        field: FieldIdx,
-        field_ty: Ty<'tcx>,
-    ) -> Expr {
-        let field_ty = self.monomorphize(field_ty);
-        if matches!(field_ty.kind(), ty::Array { .. }) {
+    fn codegen_simd_field(&mut self, parent_expr: Expr, field: FieldIdx, field_ty: Ty) -> Expr {
```

Comment:
Please let us `field_idx` for value of the `FieldIdx` type.

Reply:
sure